frameworklauncher/src/main/java/com/microsoft/frameworklauncher/webserver/LauncherModule.java

```diff
@@ -150,13 +229,41 @@ public ClusterConfiguration getClusterConfiguration(
     return requestManager.getClusterConfiguration();
   }
 
+  @PUT
+  @Path(WebStructure.ACL_CONFIGURATION_PATH)
+  @Consumes({MediaType.APPLICATION_JSON})
+  public Response putAclConfiguration(
+      @Context HttpServletRequest hsr,
```

Comment:
Any access control when putting AclConfigurations?

Reply:
Ever consider to add a initial admin user when starting Launcher? And let only admin can put AclConfigurations?

---
In reply to: [167465076](https://github.com/Microsoft/pai/pull/320#discussion_r167465076) [](ancestors = 167465076)

Reply:
Now, we trust all things from external. So, in the first stage, we can keep the simple and clear design

---
In reply to: [167465155](https://github.com/Microsoft/pai/pull/320#discussion_r167465155) [](ancestors = 167465155,167465076)